scripts/check_files_compliance.py

```diff
@@ -113,26 +112,34 @@ def ignore(root, file):
 
     # Particular cases of files that are also ignored
     if file == '.gitignore' or file == '.valgrindrc' or file == '.valgrindSuppressions' \
-        or file == 'README.md' or file == 'LICENSE' or file == 'ContributionPolicy.txt' \
-        or file == 'CHANGES_NEXT_RELEASE' or file == 'compileInfo.h' or file == 'unittests_that_fail_sporadically.txt' \
-        or file == 'Vagrantfile' or file == "contextBroker.ubuntu":
+            or file == 'README.md' or file == 'LICENSE' or file == 'ContributionPolicy.txt' \
+            or file == 'CHANGES_NEXT_RELEASE' or file == 'compileInfo.h' or file == 'unittests_that_fail_sporadically.txt' \
+            or file == 'Vagrantfile' or file == "contextBroker.ubuntu":
         return True
     if 'scripts' in root and (file == 'cpplint.py' or file == 'pdi-pep8.py' or file == 'uncrustify.cfg' \
-        or file == 'cmake2junit.xsl'):
+                                      or file == 'cmake2junit.xsl'):
         return True
     if 'acceptance' in root and (file.endswith('.txt') or file.endswith('.json')):
         return True
```

Comment:
Up to here, all the changes in check_files_compliance.py seems to be indentation... Is that correct? Why intentation has changed?